### DISCUSSION TITLE
How do I import Mail.service into Admin.controller?

### DISCUSSION THREAD

**Discussion by Gradk** [nrwl/nx#11654] [General]
I have two applications:

1) account
2) mail

I want to run a function from the first application in the second, but I get an error.

Projects cannot be imported by a relative or absolute path, and must begin with a npm scope

How do I correctly specify the path to the file?

![image](https://user-images.githubusercontent.com/50734508/185782066-1a496cbf-88e4-41e5-97f3-8a0d279fff9b.png)

**Comment by AgentEnder** [comment]
You should use the path defined in tsconfig.base.json, it likely looks something like `import ... from '@{my-org}/mail'`
